client/src/routes/HereMaps/App.css

```diff
@@ -0,0 +1,186 @@
+body{
```

Comment:
Move these changes to `HereMaps.module.scss` and use the styles variable to access styles in `index.jsx`.  See how it is already used.